src/index.html

```diff
@@ -7,8 +7,76 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="same-origin"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="same-origin"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Oswald:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#Home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a 
+              href="#" 
+              class="nav_link is-active"
+              >
```

Comment:
```suggestion
            <a 
              href="#" 
              class="nav_link is-active"
            >
```

need to fix indentation before `>` (everywhere) 😉